frontend/pages/journey/[id].vue

```diff
@@ -84,10 +84,14 @@ useHead({
 });
 
 const colorMode = useColorMode();
+const darkThemeMq = window.matchMedia("(prefers-color-scheme: dark)");
 let darkColor = "#333333";
```

Comment:
We could improve this and don't hardcode this

Reply:
then do it

Reply:
do it urself

Reply:
I don't know how, I'm just a backend worker